electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -279,6 +279,35 @@ def update_production_breakdowns(
 
         return production_breakdowns
 
+    @staticmethod
+    def filter_only_zero_production(
+        breakdowns: "ProductionBreakdownList",
+    ) -> "ProductionBreakdownList":
+        """A method to filter out production breakdowns with a total production of 0 MW."""
+        events = ProductionBreakdownList(breakdowns.logger)
+        for event in breakdowns.events:
```

Comment:
Maybe change up the naming here, I would expect `event` to be a child of `events` not `breakdown.events` at first glance.